src/charm.py

```diff
@@ -223,22 +224,20 @@ def _configure_certificate_requirers(self):
             logger.warning("couldn't distribute certificates: not logged in")
             return
         databag_csrs = self.tls.get_certificate_requests()
-        notary_table = self.client.get_certificate_requests_table(login_details.token)
-        if not notary_table:
-            logger.warning("couldn't distribute certificates: couldn't get table from notary")
-            return
-
+        certificate_requests = self.client.list_certificate_requests(login_details.token)
         for request in databag_csrs:
-            notary_rows_with_matching_csr = [
-                row
-                for row in notary_table.rows
-                if row.csr == str(request.certificate_signing_request)
+            certificate_requests_with_matching_csr = [
```

Comment:
This one was very confusing when we were discussing the PR with @saltiyazan, because there are 2 sources of CSR's here, Notary and the databags. I'm fine with it, but not sure if others would agree.

Reply:
I can go and add the `notary` prefix if it helps

Reply:
yes that would be great

Reply:
done